gw_spaceheat/README.md

```diff
@@ -11,4 +11,8 @@
   - `python -m pip install pip-tools`
   - If you want to add a new library used in all contexts, then add it to requirements/base.in and run
       - `pip-compile --output-file=requirements/dev.txt requirements/dev.in`
-    - ... and then make sure to re-compile all requirements.txt that reference that .in file (all of them,for base.in)
\ No newline at end of file
+      - ... and then make sure to re-compile all requirements.txt that reference that .in file (all of them,for base.in)
+
+**Install the mosquito server**
+1. `brew install mosquitto`
+2. `brew services restart mosquitto`
```

Comment:
Perhaps add:
3. `ln -sfv /usr/local/opt/mosquitto/*.plist ~/Library/LaunchAgents` 
(if you want to the broker to start on mac startup)
4. Test using commandline pub sub.
    - In first terminal: `mosquitto_sub -t 'test'` 
    - In second terminal: `mosquitto_pub -t 'test' -m 'hi'` 
    - Success: the subscribing terminal outputs `hi`

Reply:
added in this commit: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/5/commits/fce3b8e887a65a9ac1900481effe4d0732b3efe3